pkg/controller.go

```diff
@@ -264,7 +288,7 @@ func identifyContainersCloseToMemoryLimit(podMetrics metricsv1beta1.PodMetrics,
 		memoryUsagePercent := 100 * containerMetric.Usage.Memory().AsApproximateFloat64() / containerDefinition.Resources.Limits.Memory().AsApproximateFloat64()
 		klog.V(1).Infof("Memory usage for '%s/%s/%s' is %.1f%%\n", podMetrics.Namespace, podMetrics.Name, containerMetric.Name, memoryUsagePercent)
 
-		if memoryUsagePercent > usageMemoryUsageThresholdPercent {
+		if memoryUsagePercent >= usageMemoryUsageThresholdPercent {
```

Comment:
I felt like the expectation is for the threshold limit to be inclusive. I doubt this will break anybody's setup, unless they had the eviction limit at 100% and expected it to never fire. I would add a note in the release.